frontend/src/components/SamplesList.tsx

```diff
@@ -151,10 +151,8 @@ export default function SamplesList({
         setShowAlertModal(true);
       } else {
         const allRowNodesInView = rowNode.parent?.allLeafChildren;
-        const allRowsHaveValidCostCenter = allRowNodesInView?.every(
-          (rowNode) =>
-            rowNode.data?.costCenter &&
```

Comment:
`rowNode.data?.costCenter &&` would return `false` when running into empty values. Removed it because we do want to consider empty values in `costCenter` to be valid.